src/main/java/com/dataguise/saas/DgControllerAuthProvider.java

```diff
@@ -0,0 +1,85 @@
+package com.dataguise.saas;
+
+import com.dg.saas.orch.client.DgSecureRestDriver;
+import com.dg.saas.orch.exception.DgDrvException;
+import com.dg.saas.orch.models.structures.ErrorConstants;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.authentication.*;
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/DgControllerAuthProvider.java L8
    
    Using the &#39;.*&#39; form of import should be avoided - org.springframework.security.authentication.*.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a365276533>*